libs/platform/src/lib/approval-flow/approval-flow.component.ts

```diff
@@ -192,11 +238,16 @@ export class ApprovalFlowComponent implements OnInit, OnDestroy {
     /** @hidden */
     constructor(
         private readonly _dialogService: DialogService,
-        private readonly _messageToastService: MessageToastService,
         private readonly _cdr: ChangeDetectorRef,
+        @Optional() @Inject(DATA_PROVIDERS) private providers: Map<string, DataProvider<any>>,
         @Optional() private readonly _rtlService: RtlService
     ) {}
 
+    /** Returns snapshot of the current and initial states of approval process */
+    get approvalState(): ApprovalProcess {
```

Comment:
why do we call it `approvalState` if it returns `ApprovalProcess`? Minor but looks a little bit weird

Reply:
Yeah, I agree, will rename

Reply:
done